### ISSUE TITLE
Negative numbers in ranks causes parse error 

### ISSUE THREAD

**Issue by lemmy** [gephi/gephi#2681]
<img width="836" alt="Screenshot 2022-11-01 at 9 15 28 AM" src="https://user-images.githubusercontent.com/88777/199282585-1e35e242-064b-414f-9805-5d173257b8d0.png">

```dot
strict digraph DiskGraph {
    subgraph cluster_graph {

        -1 [label="LblA"]
        2 [label="LblB"]
        -1 -> 2

        {rank = same; -1;2;}
    }
}
```

<img width="640" alt="Screenshot 2022-11-01 at 9 12 37 AM" src="https://user-images.githubusercontent.com/88777/199282083-4ce9e2f0-1a2f-40e4-bd1c-48adf7ed18cc.png">

```

Product Version: Gephi 0.9.7 202208031831
Java: 11.0.16; OpenJDK 64-Bit Server VM 11.0.16+8
Runtime: OpenJDK Runtime Environment 11.0.16+8
System: Mac OS X version 13.0 running on x86_64; UTF-8; en_US (gephi)
User directory: /Users/markus/Library/Application Support/gephi/0.9
Cache directory: /Users/markus/Library/Caches/gephi/0.9
```

**Comment by mbastian** [comment]
Thanks @lemmy normally we allow any string for identifiers so it might be a bug. We'll look into it.

**Comment by mbastian** [comment]
Thanks for the report, fixed as part of #1075
